api/graphite.go

```diff
@@ -831,6 +831,9 @@ type consolidatorTuple struct {
 // we will collect all the individual series from the peer, and then sum here. that could be optimized
 func (s *Server) executePlan(ctx context.Context, orgId uint32, plan *expr.Plan) ([]models.Series, models.RenderMeta, error) {
 	var meta models.RenderMeta
+	traceID, _ := tracing.ExtractTraceID(ctx)
+
+	log.Infof("executePlan: Starting traceID=%s req %v", traceID, plan.Reqs)
```

Comment:
If the traceID extracted from `tracing.ExtractTraceID(ctx)` is empty, do we still want to log here?

Reply:
I think so. There's still value in the log line. Also in production this should always have a value, so it might be interesting to see if there are cases where it is empty.